include/openssl/kdf.h

```diff
@@ -69,6 +69,8 @@ void EVP_KDF_do_all_ex(OPENSSL_CTX *libctx,
 # define EVP_KDF_CTRL_SET_UKM                0x16 /* unsigned char *, size_t */
 # define EVP_KDF_CTRL_SET_CEK_ALG            0x17 /* char * */
 # define EVP_KDF_CTRL_SET_SHARED_INFO        EVP_KDF_CTRL_SET_SSKDF_INFO
+# define EVP_KDF_CTRL_SET_CIPHER             0x18 /* EVP_CIPHER */
+# define EVP_KDF_CTRL_SET_CONSTANT           0x19 /* unsigned char *, size_t */
```

Comment:
There two aren't necessary.  We shouldn't be adding CTRL functions anymore.

Reply:
Ok removed